app/javascript/packs/base.jsx

```diff
@@ -4,6 +4,35 @@ import {
   getInstantClick,
   initializeTouchDevice,
 } from '../topNavigation/utilities';
+import { fetchSearch } from '@utilities/search';
+
+// Namespace for functions which need to be accessed in plain JS initializers
+window.Forem = {};
```

Comment:
old school JS, I like it :D This reminds me also of the module pattern in jQuery :D

Reply:
We did this when the modals refactor happened so as to not pollute the `window`.